Seems like --yes stopped working

### Describe the bug

We are using the `--yes` option for publishing tasks, which worked in 1.13. but seems to be ignored in 1.15.x.

CI Job:

```
> new-publish:ci
> lerna publish --yes

lerna-lite notice cli v1.15.1
lerna-lite info ci enabled
lerna-lite info current project version 0.8.21
lerna-lite info Assuming all packages changed
? Select a new version (currently 0.8.21) (Use arrow keys)
❯ Patch (0.8.22) 
  Minor (0.9.0) 
  Major (1.0.0) 
  Prepatch (0.8.22-alpha.0) 
  Preminor (0.9.0-alpha.0) 
  Premajor (1.0.0-alpha.0) 
  Custom Prerelease 
  Custom Version 
Error: Process completed with exit code 130.
```

Corresponding npm script:

```
"new-publish:ci": "lerna publish --yes",
```

### Expectation

--yes should work

### Reproduction

Run `lerna publish --yes` in a CI environment, e.g.: Github Actions

### Lerna config and logs

<details><summary>lerna.json</summary><p>
<!-- browsers demand the next line be empty -->

```json
{
  "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
  "packages": [
    "packages/@pandino/*"
  ],
  "npmClient": "npm",
  "version": "0.8.21",
  "ignoreChanges": [
    "**/__fixtures__/**",
    "**/__tests__/**",
    "**/helpers/**",
    "**/*.md"
  ]
}

```
</p></details>

<details><summary>lerna-debug.log</summary><p>
<!-- browsers demand the next line be empty -->

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>


### Environment Info

lerna-lite version: 

```
"@lerna-lite/cli": "^1.15.1",
"@lerna-lite/run": "^1.15.1",
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/lerna-lite/lerna-lite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/lerna-lite/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/lerna-lite/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/lerna-lite/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/README.md
+++ b/README.md
@@ -120,7 +120,7 @@ changes and changelogs
 
 #### [Version](https://github.com/lerna-lite/lerna-lite/tree/main/packages/version) and [Publish](https://github.com/lerna-lite/lerna-lite/tree/main/packages/publish) commands (included with the CLI)
 
-- Automate the rolling of new Versions (`independent` or `fixed`) of all your workspace packages.
+- Automate the rolling of new Versions (`independent` or fixed version) of all your workspace packages.
   - it will automatically Commit/Tag your new Version & create new GitHub/GitLab Release (when enabled).
 - Automate, when enabled, the creation of Changelogs for all your packages by reading all [Conventional Commits](https://www.conventionalcommits.org/) and a merged one in the project root.
 - Automate, when enabled, the repository Publishing of your new version for all your packages (NPM or other platforms).
@@ -176,6 +176,8 @@ Note that `package-a` will not be created, it is only shown here to help clarify
 Run the following commands to install Lerna-Lite in your project and/or install it globally by adding the `-g` option.
 
 If you are new to Lerna-Lite, you could also run the [lerna init](https://github.com/lerna-lite/lerna-lite/tree/main/packages/init#readme) command which will create the `lerna.json` for you with a minimal setup. If you are using a different client other than npm, then make sure to update the `npmClient` property in `lerna.json` (for example: `"npmClient": "yarn"` or `"pnpm"`).
+
+> **Note** please make sure that you have a `lerna.json` config file created and a `version` property defined with either a fixed or independent mode. An error will be thrown if you're missing any of them.
 
 ### JSON Schema
 You can add the `$schema` property into your `lerna.json` to take advantage of Lerna-Lite [JSON Schema](https://json-schema.org/) (`lerna init` will automatically configure this for you). This will help with the developer experience, users will be able to see what properties are valid with a brief description of what they do (each description are pulled from their associated lerna command options documentation).

--- a/packages/core/src/command.ts
+++ b/packages/core/src/command.ts
@@ -288,7 +288,10 @@ export class Command<T extends AvailableCommandOption> {
     }
 
     if (!this.project.version) {
-      throw new ValidationError('ENOVERSION', 'Required property version does not exist in `lerna.json`');
+      throw new ValidationError(
+        'ENOVERSION',
+        'Required property `version` does not exist in `lerna.json`, make sure to provide one of two modes (fixed or independent). For example "version": "independent" OR "version": "1.0.0"'
+      );
     }
 
     if ((this.options as InitCommandOption).independent && !this.project.isIndependent()) {

--- a/packages/version/README.md
+++ b/packages/version/README.md
@@ -20,6 +20,8 @@ npm install @lerna-lite/cli -D -W
 lerna version
 ```
 
+> **Note** please make sure that you have a `lerna.json` config file and a `version` property defined with either a fixed or independent mode (for example: `"version": "independent"`). An error will be thrown if you're missing any of them.
+
 ## Usage
 
 ```sh

